applications/crossbar/src/modules/cb_accounts.erl

```diff
@@ -1342,8 +1342,7 @@ set_notification_preference(Context, Preference) ->
 
 -spec create_account_mod(ne_binary()) -> any().
 create_account_mod(AccountId) ->
-    Db = kz_util:format_account_mod_id(AccountId),
-    kazoo_modb:create(Db).
+    kazoo_modb:create(kazoo_modb:get_db(AccountId)).
```

Comment:
it's related to wht_util, `kazoo_modb:create` was expecting raw account_modb which would be passed to create_routines